### DISCUSSION TITLE
퓨전 개발자 스터디 후기

### DISCUSSION THREAD

**Discussion by coogie02** [fusiondevkr/pl900#59] [1 Q&A]
안녕하세요...
6주간의 스터디 일정으로 리더님 그리고 참석자분들 모두 고생하셨습니다.
감사합니다.
https://blog.naver.com/coogie02/222489427601

**Comment by justinyoo** [comment]
고생하셨습니다! 다음 과정에서 다시 만나요!

**Comment by sesme100** [comment]
확인하였습니다! 진심으로 담긴 후기 감사합니다.
